src/java.base/share/classes/java/lang/foreign/Arena.java

```diff
@@ -31,47 +31,53 @@
 import java.lang.foreign.MemorySegment.Scope;
 
 /**
- * An arena controls the lifecycle of native memory segments, providing both flexible allocation and timely deallocation.
+ * An arena controls the lifecycle of native memory segments, providing both flexible
+ * allocation and timely de-allocation.
```

Comment:
Please revert all uses of "de-allocation". I don't think that's incorrect - seems more of a pedantic check from the tool.